docs/release-train-issue-template.md

```diff
@@ -54,14 +48,14 @@ Variables to be expanded in this template:
 
 ### Announcements
 
-- [ ] For non-patch releases: Merge draft news item for [akka.io](https://github.com/akka/akka.github.com)
+For non-patch releases:
```

Comment:
Does this imply no announcing at all for patch releases?

Reply:
Optionally - though if there's something notable to mention, perhaps it should be a minor release anyway (at least until we hit 1.0.0)